src/index.ts

```diff
@@ -15,7 +15,12 @@ export type KeyDerivationOptions = {
 
 export type EncryptionKey = {
   key: CryptoKey;
-  derivationFunction: KeyDerivationOptions;
+  derivationFunction?: KeyDerivationOptions;
```

Comment:
this is optional as a user may have a key without knowing how it was derived, but nevertheless, it should still be usable to decrypt / encrypt data